package.json

```
@@ -71,7 +71,7 @@
   ],
   "dependencies": {},
   "peerDependencies": {
-    "react-native": ">=0.59.0"
+    "react-native": ">=0.60.2"
```

There is an unrelated issue that makes this better as an even higher peer dependency now:
```suggestion
    "react-native": ">=0.63.3"
```

(back story: Xcode 12.5 broken react-native and 0.63.3 is the oldest version patched so it's the oldest version still compatible with Xcode really)